io.openems.edge.timeofusetariff.entsoe/src/io/openems/edge/timeofusetariff/entsoe/BiddingZone.java

```diff
@@ -0,0 +1,44 @@
+package io.openems.edge.timeofusetariff.entsoe;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum BiddingZone implements OptionsEnum {
```

Comment:
There is no need for BiddingZone to be an `OptionsEnum`. A plain `enum` is sufficient, and you won't need `UNDEFINED` then.